src/ansys/units/unit.py

```diff
@@ -0,0 +1,212 @@
+import ansys.units as ansunits
+from ansys.units.utils import filter_unit_term
+
+
+class Unit:
+    """
+    Initializes a Unit. Contains all the unit information.
+
+    Parameters
+    ----------
+    Name: str,
+        Name of the unit or string chain of combined units
+    Config: dict
+        dictionary of unit properties
+    Dimensions: list
+        A list of length 9 to describe all base units.
```

Comment:
I suppose it would be length 10 but maybe its best to not refer to the length

Reply:
I changed it to 10. If it is more then 10 an error is raised. What would the benefit of not saying the length be?

Reply:
That said.  The Unit should be taking a `Dimensions` object not a list of dimensions. :-)  In that case the documentation here will be different.

Reply:
Ok I will make those changes.

Reply:
The arbitrary dimensions `list` is a bit of a concern. We can be quite type-safe with `dict`s combined with constants exposed by the `Dimensions` class.